index.html

```diff
@@ -174,15 +174,15 @@ <h3 class="styleh3">CONTACT US</h3>
     <p class="division_sections"></p>
     <footer class="footer_class">
       <div class="logos">
-        <img src="img/Vector.png" alt="GitHub">
-        <img src="img/dos.png" alt="LinkedIn">
-        <img src="img/instagram.png" alt="Instagram">
-        <img src="img/cuatro.png" alt="Medium">
+        <img src=".img/Vector.png" alt="GitHub">
+        <img src=".img/dos.png" alt="LinkedIn">
+        <img src=".img/instagram.png" alt="Instagram">
+        <img src=".img/cuatro.png" alt="Medium">
       </div>
       <p class="emails">businesswitharmando@gmail.com</p>
     </footer>
     <p class="division_footer"></p>
   </div>
-  <script src="/javascript.js"></script>
```

Comment:
### Your Design
![navbar](https://user-images.githubusercontent.com/93039359/179649780-bff4f8f8-3341-4f81-beb4-c8bb811f0f10.png)

### Figma Template
### 1. Fix
![figma-nav (copy)](https://user-images.githubusercontent.com/93039359/179649813-55301362-376a-424e-a810-c9d6ed04cb79.png)

### 2. Fix
![figma-nav (another copy)](https://user-images.githubusercontent.com/93039359/179649848-076221aa-e668-4053-a9ea-fdb79c520cd5.png)

- [ ] It's great that you have implemented a navbar in your markup file. :blush: But the design is not following the Figma template :cry: Please make your design follow the Figma template :+1: